www/index.html

```diff
@@ -67,17 +67,25 @@
 
     <style is="custom-style">
         paper-toolbar.custom {
-            --paper-toolbar-background: green;
+            --paper-toolbar-background: var(--paper-green-700);
```

Comment:
I'm not sure that css variables will be supported in our webview (Crosswalk) 
Maybe u can find a compatiblity table somewhere.